cmake/SetDefaults.cmake

```diff
@@ -116,19 +116,30 @@ if(DEFINED CMAKE_CXX_COMPILER)
     find_package(Boost REQUIRED regex system filesystem program_options)
 endif()
 find_package(Git REQUIRED)
-foreach(SCHE_LIBS glib-2.0 gthread-2.0 gio-2.0 gobject-2.0)
+
+# libmagic, libgcrypt does not have pc module on Debian buster
+# json-c does not have cmake aware packages on Debian buster
+foreach(SCHE_LIBS
+        glib-2.0 gthread-2.0 gio-2.0 gobject-2.0 rpm libxml-2.0 libxslt icu-uc
+        json-c
```

Comment:
Two packages have been removed but only one was added back, is the other one not required?

Reply:
`jsoncpp` has been added to copyright package as currently it only uses it.